src/node_crypto.cc

```diff
@@ -3952,6 +3954,37 @@ void SignBase::CheckThrow(SignBase::Error error) {
   }
 }
 
+int SignBase::GetRSAOptions(Environment* env, v8::Local<v8::Object> options,
```

Comment:
@bnoordhuis I can fix this by adding more positional arguments if you think that is the reasonable way. I do not know enough about node internals to fully evaluate the effects. I guess I should move default values to JS as well, such that the native code does not need to handle undefined values.

I am still a little concerned considering that it might be necessary to add more options in the future and all of these are optional, meaning that we might end up passing useless arguments most of the time.